src/pages/patientView/clinicalInformation/PatientViewPageStore.ts

```diff
@@ -388,6 +388,17 @@ export class PatientViewPageStore {
         }
     });
 
+    readonly studyToGeneticProfileDiscrete = remoteData({
+        await: ()=>[this.geneticProfileIdDiscrete],
+        invoke:async ()=>{
+            const ret:{[studyId:string]:GeneticProfile} = {};
+            if (this.geneticProfileIdDiscrete.result) {
+                ret[this.studyId] = await client.getGeneticProfileUsingGET({geneticProfileId:this.geneticProfileIdDiscrete.result});

```

Comment:
not sure if i understand.  wont this map only ever have one key?  the studyId?

Reply:
thats right, but this is just to fit the interface of DiscreteCNACache